spec/datetimeformat.html

```diff
@@ -94,9 +94,18 @@ <h1>
           1. Set _timeZone_ to DefaultTimeZone().
         1. Else,
           1. Set _timeZone_ to ? ToString(_timeZone_).
-          1. If the result of IsValidTimeZoneName(_timeZone_) is *false*, then
-            1. Throw a *RangeError* exception.
+        1. If IsTimeZoneOffsetString(_timeZone_) is *true*, then
+          1. Let _parseResult_ be ParseText(StringToCodePoints(_timeZone_), |UTCOffset|).
+          1. Assert: _parseResult_ is a Parse Node.
+          1. If _parseResult_ contains more than one |MinuteSecond| Parse Node, throw a *RangeError* exception.
+          1. Let _offsetNanoseconds_ be ParseTimeZoneOffsetString(_timeZone_).
+          1. Let _offsetMinutes_ be _offsetNanoseconds_ / (6 × 10<sup>10</sup>).
+          1. Assert: _offsetMinutes_ is an integer.
+          1. Set _timeZone_ to FormatOffsetTimeZoneIdentifier(_offsetMinutes_).
```

Comment:
I can no longer find FormatOffsetTimeZoneIdentifier in ECMA262 now.

Reply:
What I mean is if I have the following
```
let output = (new Intl.DateTimeFormat("en", {timeZone})).resolvedOptions().timeZone
```

If timeZone is "+00:00" "-0000", it will hard for us to get it back as "+00:00" "-0000" in v8 now. Since we depends on information inside ICU to be efficiently hold the nessary and useful information the output will all be "UTC" for them.  I see no practical reason why these could not be fold into "UTC"

Reply:
> I can no longer find FormatOffsetTimeZoneIdentifier in ECMA262 now.

This AO is only in Temporal (https://tc39.es/proposal-temporal/#sec-temporal-formatoffsettimezoneidentifier) and in this PR. This AO serves two purposes:
* Formatting offset IDs when they're needed, e.g. in the TimeZone.p.id getter
* When paired with ID parsing, enables normalization of offset time zone IDs

Once Temporal makes it to Stage 4, we'd file another 402 PR to remove FormatOffsetTimeZoneIdentifier from 402 as part of the refactor described in https://github.com/tc39/ecma402/pull/788#discussion_r1320356161.

But in the meantime, we need it here to support offset time zone ID normalization.

Reply:
Sorry. My bad, I thought FormatOffsetTimeZoneIdentifier is in ECMA262 and forget to search this PR.